src/pages/instances/InstanceOverview.tsx

```diff
@@ -44,7 +44,7 @@ const InstanceOverview: FC<Props> = ({ instance }) => {
       />
       <Row className="general">
         <Col size={3}>
-          <h2 className="p-heading--4">General</h2>
+          <h2 className="p-heading--5">General</h2>
```

Comment:
I think this requires design input. I don't see a valid reason to suddenly change this style.

Reply:
@piperdeck can you have a look at this? Is the change to new layout (bolded section title in the same font size as the rest of the content) from the old layout (bigger font for the section, not bold) acceptable?